src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -186,10 +187,9 @@ import { Integrations as ApmIntegrations } from '@sentry/apm';
 Sentry.init({
     dsn: '___PUBLIC_DSN___',
     integrations: [
-        new Integrations.Tracing({
-            tracesSampleRate: 0.1,
-        }),
+        new ApmIntegrations.Tracing(),
     ],
+    tracesSampleRate: 0.1,
```

Comment:
@HazAT Just curious, is this the preferred placement of the configuration? Wondering because of https://github.com/getsentry/sentry/blob/3672ca751d9306fc22761f1a52847787fcef7915/src/sentry/static/sentry/app/bootstrap.jsx#L37-L40

Reply:
This will be fixed in the next update, should have never been there in the first place.

Reply:
Gotcha. Thanks for clarifying 👍